src/Game.ts

```diff
@@ -0,0 +1,103 @@
+export class Game {
+    private _lastSymbol: string = ' ';
+    private _board: Board = new Board();
+
+    public Play(symbol: string, x: number, y: number) : void {
+        //if first move
+        if (this._lastSymbol == ' ') {
+            //if player is X
+            if (symbol == 'O') {
+                throw new Error("Invalid first player");
+            }
+        }
+        //if not first move but player repeated
+        else if (symbol == this._lastSymbol) {
+            throw new Error("Invalid next player");
+        }
+        //if not first move but play on an already played tile
+        else if (this._board.TileAt(x, y).Symbol != ' ') {
```

Comment:
This is an example of content coupling. You're dependent on the boards internal elements